### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by NRothera**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+#bomberman.left {background-position: -87px 0px;}
+#bomberman.left-one {background-position: -101.5px 0px;}
+#bomberman.left-two {background-position: -116px 0px;}
+
+#bomberman.right {background-position: -130.5px 0px;}
+#bomberman.right-one {background-position: -145px 0px;}
+#bomberman.right-two {background-position: -159.5px 0px;}
+
+/*#bomberman2{
```
Comment (JSChilds) [inline css/styles.css]: Don't leave commented-out code in a project you are submitting as finished.

File: js/player2.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+// var currentKey2;
```
Comment (JSChilds) [inline js/player2.js]: Don't include files in the file system that you aren't using as part of this delivered version.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Remove the commented-out code and unused player2.js file and you will be ready to merge.

**Comment by NRothera** [comment]
Yeah sorry I saw he said that after I submitted it! My bad

**Review by j-mcgregor — APPROVED** [review]
Very clean, well done

**Review by JSChilds — APPROVED** [review]

**Pull request merged**